generate_manifest.py

```diff
@@ -77,6 +76,4 @@ def required_to_deploy_this_project(project, specific_projects):
                 project_detail['Packages'] = octo.get_latest_packages(project_id)
             manifest['Projects'][project] = project_detail
 
-    pprint(manifest)
-    with open('manifest.json', 'w') as outfile:
-        json.dump(manifest, outfile)
\ No newline at end of file
+    sys.stdout.write(json.dumps(manifest, indent=1))

```

Comment:
I wondered about that one. TBF I may have caused confusion by suggesting `json.dumps` w/indent. could just `pprint` with indent instead...

Reply:
There seem to be quite a few posts / articles suggesting `print('abc')` does almost the same as `sys.stdout.write('abc')` the only difference seems to be that `print` adds a new line to the stdout.
If you can get `pprint` to work I would keep with that, as it will make the files much easier to read.

Reply:
You may want to use 
`
pp = pprint.PrettyPrinter(indent=4)
pp.pprint(obj)
`

Reply:
So I'm going for `print(json.dumps(manifest, indent=1))`, which still pretty prints because of the `indent=1` and `json.dumps` doesn't play nicely with `pprint` (and `pprint` by itself doesn't emit valid JSON)